From 392ee749a39bcc3522de55dd4e5d97c39cf4c845 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Fri, 16 Jan 2015 21:11:37 -0800
Subject: [PATCH 3006/3008] buildit: Set compatibility version to
 RC_ProjectSourceVersion

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit fb96b42906511e4afa3c1bcf4f4be47e87831137)
---
 lib/buildit | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git libcxx-6.0.0.src/lib/buildit libcxx-6.0.0.src/lib/buildit
index 0a585022a..4fd356420 100755
--- libcxx-6.0.0.src/lib/buildit
+++ libcxx-6.0.0.src/lib/buildit
@@ -59,7 +59,8 @@ case $TRIPLE in
     then
         EXTRA_FLAGS="-nostdinc++ -std=c++11 -U__STRICT_ANSI__"
         LDSHARED_FLAGS="-o libc++.1.dylib \
-            -dynamiclib -nodefaultlibs -current_version 1 \
+            -dynamiclib -nodefaultlibs \
+            -current_version ${RC_ProjectSourceVersion} \
             -compatibility_version 1 \
             -install_name /usr/lib/libc++.1.dylib \
             -Wl,-reexport_library,/usr/lib/libc++abi.dylib \
-- 
2.15.1

